python/lsst/ts/observatory/control/remote_group.py

```diff
@@ -372,52 +479,117 @@ async def check_comp_heartbeat(self, component):
         """
 
         while True:
-            await getattr(self.rem, component).evt_heartbeat.next(
+            await getattr(self.rem, self._components[component]).evt_heartbeat.next(
                 flush=True, timeout=self.fast_timeout
             )
 
+    async def inspect_settings(self, components=None):
+        """Return available settings.
+
+        Inspect `settingVersions` from CSCs and return dictionary with all the
+        available settings.
+
+        Parameters
+        ----------
+        components : `list` of `str` or `None`
```

Comment:
The doc string says that the names must be component attribute names, though it also talks about the `components` property.

Please rename this argument to components_attr if that's what it is, or fix the doc string, if not.